source/sessions/driver-sessions.rst

```diff
@@ -305,8 +306,8 @@ This property returns the ``SessionOptions`` that were used to start this
 sessionId
 ---------
 
-This property returns the session ID of this session. Note that if server
-sessions are pooled, different ``ClientSession`` instances will have the same session ID,
+This property returns the session ID of this session. Note that since ``ServerSession``s
+are pooled, different ``ClientSession`` instances will have the same session ID,
```

Comment:
This line does not render correctly. Also:
"instances will have" -> "instances can have"

Reply:
fixed the rendering here, and language